### DISCUSSION TITLE
记事本🗒️

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#759] [General]
https://github.com/Advanced-Frontend/Daily-Interview-Question/issues/87

**Comment by mark9804** [comment]
老大哥要听 Ruby on rails 微服务通用架构设计的东西吗，我之前偶然做东西认识了个 ruby 方向的 leader ，虽然人这个分享是卖课，但前面的架构设计不知道有没有帮助



<details>
<img src="https://user-images.githubusercontent.com/9006264/164608442-3e2e2848-0543-47e0-824a-90a67d36d951.jpeg" width="300" />

<img src="https://user-images.githubusercontent.com/9006264/164608384-bb5e1960-60e9-44bd-9719-63bfce8a17f9.jpeg" width="300" />
</details>

**Reply by Arondight** [reply]
谢谢，有空就听一下

**Comment by mark9804** [comment]
问了一下项目里几个工作时间不是非常长的前端，都表示自己都是直接 post 的，没听说过埋点
